packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetailRow.test.js

```diff
@@ -20,6 +20,8 @@ import SpanDetail from './SpanDetail';
 import DetailState from './SpanDetail/DetailState';
 import SpanTreeOffset from './SpanTreeOffset';
 
+jest.mock('./SpanTreeOffset');
```

Comment:
Nice, I thought this needed to go above the imports :+1:

Reply:
I maybe I did that out of convention?

Reply:
moved `jest.mock` to the top and confirmed tests still pass, but eslint complains. I am going to keep where it is for now.